Gemfile.lock

```diff
@@ -549,7 +549,7 @@ DEPENDENCIES
   yt (~> 0.30.1)
 
 RUBY VERSION
-   ruby 2.3.1p112
+   ruby 2.4.0p0
```

Comment:
Can you do it in separate PR, because we need to update ruby version on `production` before doing that. Also it is not related to `ui-fixes`